populate_dev_data_handler.go

```diff
@@ -21,6 +21,8 @@ import (
 
     "google.golang.org/appengine"
     "google.golang.org/appengine/datastore"
+    "errors"
```

Comment:
Can you move this up above the `google.golang` imports?

Reply:
Done.